src/Compilers/CSharp/Portable/Binder/SubsumptionDiagnosticBuilder.cs

```diff
@@ -129,6 +129,8 @@ private ErrorCode CheckSubsumed(BoundPattern pattern, DecisionTree decisionTree,
                         // the input can no longer be null. In that case a null pattern is subsumed.
                         if (isNull && !inputCouldBeNull)
                         {
+                            // Note: we do not have any test covering this. Is it reachable?
```

Comment:
The `if` is there because of the expectation that this code path will come into play as the set of supported patterns is expanded.

---
In reply to: [100589351](https://github.com/dotnet/roslyn/pull/16928#discussion_r100589351) [](ancestors = 100589351)